deploy/helpers/liquityMocks.ts

```diff
@@ -68,3 +68,7 @@ export default async function (env: HardhatRuntimeEnvironment, prefix: string) {
     );
   }
 }
+
+liquityMocks.skip = async (hre: HardhatRuntimeEnvironment) => true;
```

Comment:
I don't understand. Why do helpers need this? I don't think I needed it when I introduced them

Reply:
When you run the entire deployment and that is not there, hardhat executes the function with hre and then undefined for the rest of the arguments

All default exports within the deployments folder are executed by the deployment suite when not filtering by tag

Reply:
I see. I was only running with some tags. I don't like this at all, these were supposed to be just plain, simple functions. But oh well